.travis.yml

```diff
@@ -1,46 +1,41 @@
-#Trying to go to the faster container infrastructure of github
+dist: trusty
 sudo: false
-
 language: php
 
 php:
-  - 5.6
-  - 7.0
   - 7.1
-  - nigthly
```

Comment:
That error seems unrelated, looks like you force-pushed the branch before Travis was able to grab the commit. :P

> the pipeline didn't work even though that one as allowed to fail

Maybe there's a difference betwen `errored` and `failed` state.